test/Vault.spec.ts

```diff
@@ -905,10 +1094,19 @@ describe("Vault", () => {
       });
 
       await vault.connect(alice).deposit(params);
-      await addYieldToVault("100");
-
-      expect(await vault.yieldFor(alice.address)).to.eq(parseUnits("50"));
-      expect(await vault.yieldFor(bob.address)).to.eq(parseUnits("50"));
+      const newYield = await addYieldToVault("100");
```

Comment:
I guess this variable could be just `yield`?

Reply:
It is a new generated yield.
yield is a total yield.
In this test case, we added yield one time, so newYield = total yield, however newYield is more make sense to me.

Reply:
and `yield` is a typescript keyword which we cannot use as variable